internal/services/notification/db/objects/objects.go

```diff
@@ -0,0 +1,34 @@
+package objects
+
+import (
+	"agola.io/agola/internal/sqlg"
+)
+
+const (
+	Version = uint(1)
+)
+
+const TypesImport = "agola.io/agola/services/notification/types"
+
+var AdditionalImports = []string{"time"}
+
+var ObjectsInfo = []sqlg.ObjectInfo{
+	{
+		Name: "RunWebhook", Table: "runwebhook",
+		Fields: []sqlg.ObjectField{
+			{Name: "Payload", Type: "string"},
+		},
+	},
+	{
+		Name: "RunWebhookDelivery", Table: "runwebhookdelivery",
+		Fields: []sqlg.ObjectField{
+			{Name: "Sequence", Type: "uint64", Sequence: true},
+			{Name: "RunWebhookID", Type: "string"},
+			{Name: "DeliveryStatus", Type: "types.DeliveryStatus", BaseType: "string"},
+			{Name: "DeliveredAt", Type: "time.Time", Nullable: true},
+			{Name: "Duration", Type: "time.Duration", Nullable: true},
+			{Name: "Status", Type: "string"},
+			{Name: "StatusCode", Type: "int"},
```

Comment:
you probably need a foreign key to RunWebhook.

Reply:
We have `{Name: "RunWebhookID", Type: "string"},`  at row 26 is not good?

Reply:
I meant you should add a relation between RunWebhookDelivery and RunWebhook.